index.html

```diff
@@ -69,8 +73,124 @@
 
     <section class='informative'>
       <h2>Introduction</h2>
-      <p>This document is a stub.</p>
+      <p>The Browser Extensions Community Group's goal is to make browser extension code much more interoperable across browsers by specifying common extension interfaces and well-defined browser behavior. </p>
+      <p>This specification defines the interfaces and behavior which enables extensions to communicate with local client native processes outside of the browser environment.</p>
+    </section>
+
+    <section class='informative'>
+        <h2>Core APIs</h2>
+        <p>The goal for these APIs is to provide functionality that extension authors need for:</p>
+        <ul>
+            <li>message-passing between extensions and local client processes outside of the browser</li>
+            <li>message-passing to long-lasting connection-based ports</li>
+            <li>message-passing to connectionless processes</li>
+        </ul>
+        <section>
+            <h4 id="runtime">The <code>runtime</code> object</h4>
+            <section class="introductory">
+                <h4>Overview</h4>
+                <p>
+                    In addition to the base functionality of the <code>browser.runtime</code> object defined in the <a href="https://browserext.github.io/browserext#runtime">Browser Extensions specification</a>, these additional APIs provide functionaly for Native Messaging.
+                </p>
+            </section><section class="introductory">
+                <h4>Manifest</h4>
+                <p>
+                    The <code>"nativeMessaging"</code> permission is required to use these additional APIs.
+                </p>
+<pre>
+{
+    "permissions": [                               // Required
+        "nativeMessaging"                          // Required
+    ]                                              // Required   
+}
+</pre>
+            </section>
+            <section class="introductory">
+                <h4>Context</h4>
+                <p>
+                    This API is available in Window context (background, event, popup, options and custom pages), Content Script context, and Internet-hosted web page context.
```

Comment:
In Firefox, native messaging is only available from background pages, and I'm pretty sure the same is true in Chrome.  I think it makes sense to specify it that way here.